test/studies/twitter/sleepsort.chpl

```diff
@@ -0,0 +1,11 @@
+use Time;
```

Comment:
I have a feeling this program might turn into a maintenance headache when some system decides that sleeping for 3 seconds is a fine reaction to a sleep(1) call.  Hopefully not though.